contracts/GringottsBank.sol

```diff
@@ -129,6 +131,30 @@ contract  GringottsBank is DSAuth, BankSettingIds {
         }
     }
 
+     function burndrop(uint256 _depositID, address _depositor, uint48 _months, uint48 _startAt, uint64 _unitInterest, uint128 _value, bytes _data) public {
```

Comment:
The convention is to place these changes and new file to a new logic contract e.g. GringottsBankV2.sol